src/Sylius/Bundle/AdminBundle/templates/order/history/common/sections/addresses/address/details/customer_profile.html.twig

```diff
@@ -0,0 +1,8 @@
+{% set log = hookable_metadata.context.log %}
+
+{%- for field, data in log.data|filter(data => not data is empty) -%}
+    <tr>
+        <td class="col-4 border-0"><strong>{{ ('sylius.ui.' ~ field|u.snake)|trans }}</strong></td>
+        <td class="col-8 border-0 text-body w-full"><span>{{ data }}</span></td>
+    </tr>
+{%- endfor -%}
```

Comment:
Should be further split into smaller hooks to allow inserting possible customizations between these fields.